core/tests/protractor_desktop/additionalPlayerFeatures.js

```diff
@@ -260,8 +260,8 @@ describe('Full exploration editor', function() {
     explorationEditorMainTab.setContent(forms.toRichText(
       'Parent Exploration Content'));
     explorationEditorMainTab.setInteraction(
-      'MultipleChoiceInput',
-      [forms.toRichText('Correct'), forms.toRichText('Incorrect')]);
+      'MultipleChoiceInput', [{ editAction: 'create'},
```

Comment:
There's an extra space, and it would be more readable if the `{editAction...}` were on a new line. That makes the outer list clearer

```suggestion
      'MultipleChoiceInput', [
        {editAction: 'create'},
```

Reply:
done, I've changed this code